packages/components/src/ActionList/ActionList.tsx

```diff
@@ -99,6 +107,7 @@ export interface ActionListProps {
    * Options for bulk actions. Having a non-null bulk prop will auto-enable an Action List's control bar
    */
   bulk?: BulkActionsConfig
+  filters?: FiltersConfig
```

Comment:
- Separate responsibilities for filters vs column selection.
- Rename `columnsCustomizable` to more explicitly name `canSelectDisplayedColumns`

```suggestion
  /** Add description here **/
  filters?: FiltersConfig
  /** Add description here **/
  canSelectDisplayedColumns?: boolean
```